### PR TITLE
Added reference to macro to en and fr descriptions of specDesc for #2061

### PR DESCRIPTION
**Pull request by JanelleJenstad**
@ebeshero and I got started on this ticket by changing the wording as requested (in both en and fr; we consulted @MarjorieBurghart to confirm @JanelleJenstad's French translation). We did this on a branch because we anticipate more work based on @sydb's comments on ticket #2061.

### CODE REVIEW

**Comment by ebeshero** [comment]
@sydb @martinascholger @hcayless I believe that @JanelleJenstad and I have made the small changes requested for #2061. We request a quick review to see if this is ready to pull in to dev branch in time for the upcoming 4.2.0 release. 

(There's more work perhaps to be done for #2061 down the road, but this PR is just a start to clarify the wording, as originally requested in the ticket. )

File: P5/Source/Specs/specDesc.xml [hunk 1]
```diff
@@ -13,18 +13,18 @@ $Id$
   <gloss versionDate="2007-05-02" xml:lang="zh-TW">元素或元素集描述</gloss>
   <gloss versionDate="2007-06-12" xml:lang="fr">specification description</gloss>
   <gloss versionDate="2007-05-04" xml:lang="es">descripción de elemento o clase.</gloss>
-  <gloss versionDate="2007-01-21" xml:lang="it">descrizione di elemento o classe</gloss>
-  <desc versionDate="2005-08-06" xml:lang="en">indicates that a description of the specified element or class should be included at this
+  <gloss versionDate="2007-01-21" xml:lang="it">descrizione di elemento, macro o classe</gloss>
```
Comment (sydb) [inline P5/Source/Specs/specDesc.xml]: `@versionDate` should be today, no?

**Review by sydb — COMMENTED** [review]
Except for the one comment, below, looks good to my monolingual eyes :-)

**Comment by sydb** [comment]
The `@versionDate` perhaps should have been 2021-01-29, not today, but there is an argument to be made either way, and it does not make one whit of difference. Thus going to merge this puppy … [as soon as local build tests pass]

**Review by martinascholger — APPROVED** [review]

**Pull request merged**